Framework/Algorithms/src/FitPeaks.cpp

```diff
@@ -431,6 +433,10 @@ void FitPeaks::init() {
                   "Used for validating peaks before fitting. If the total peak window Y-value count "
                   "is under this value, the peak will be excluded from fitting and calibration.");
 
+  declareProperty(PropertyNames::PEAK_MIN_SIGNAL_TO_SIGMA_RATIO, 0.,
+                  "Used for validating peaks before fitting. If the signal-to-sigma ratio is under this value, "
```

Comment:
```suggestion
                  "Used for validating peaks after fitting. If the signal-to-sigma ratio is under this value, "
```
Sorry one last thing, just spotted this typo!